### PR TITLE
New Hos Clothing/Fixed Alinement Issues

### PR DESCRIPTION
**Pull request by Catdere**
-adds: 8 new clothing items 
-fixes: clothing sprites that would off centered should be fixed now
-rebalance: Head of Security now has access to the Hudpatch a combination of the Sechud and Sec-Glasses
-rebalance Head of Personnel Starts with their own armored coat

### CODE REVIEW

**Comment by Catdere** [comment]
spooky felinid error

File: Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml [hunk 1]
```diff
@@ -51,10 +51,11 @@
...
   id: HoPGear
   equipment:
-    jumpsuit: ClothingUniformJumpsuitHoP
+    jumpsuit: ClothingUniformJumpsuitHopAlt
     back: ClothingBackpackHOPFilled
-    shoes: ClothingShoesColorBrown
-    head: ClothingHeadHatHopcap
+    shoes: ClothingShoesLeather
+    head: ClothingHeadHopCapAlt
+    outerClothing: ClothingOuterArmorHopJacket
```
Comment (VasilisThePikachu) [inline Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml:58]: No change this

File: Resources/Prototypes/Roles/Jobs/Security/head_of_security.yml [hunk 2]
```diff
@@ -38,9 +38,9 @@
     jumpsuit: ClothingUniformJumpsuitHoS
     back: ClothingBackpackHOSFilled
     shoes: ClothingShoesBootsCombatFilled
-    outerClothing: ClothingOuterCoatHoSTrench
-    eyes: ClothingEyesGlassesSecurity
-    head: ClothingHeadHatBeretHoS
+    outerClothing: ClothingOuterCoatHosjacket
+    eyes: ClothingEyesSolidEye
+    head: ClothingHeadHatSyndieMAA
```
Comment (VasilisThePikachu) [inline Resources/Prototypes/Roles/Jobs/Security/head_of_security.yml]: and tis

**Review by VasilisThePikachu — CHANGES_REQUESTED** [review]
Don't change default clothes, provide them in a closet/vending machine etc

**Review by VasilisThePikachu — CHANGES_REQUESTED** [review]
Don't change default clothes, provide them

**Comment by VasilisThePikachu** [comment]
Error is yours btw, after that i think we can merge

**Comment by VasilisThePikachu** [comment]
Make separate PR's you keep putting in stuff unrelated to this pr and need separate reviews

**Pull request closed without merging**